src/core/processing-queue.js

```diff
@@ -21,69 +20,91 @@ import {
 
 let priorityCount = 0;
 let unitSampler;
+const taskQueue = [];
```

Comment:
It would be nice to have a comment describing what a `task` is within the `taskQueue`. Something to the effect of:

> This is queue of functions that are tasks within a single test. After tests and removed from config.queue they are expanded into a set of tasks in this queue.